mm2src/coins/utxo/rpc_clients/connection_manager_multiple.rs

```diff
@@ -297,7 +296,7 @@ impl ConnectionManagerMultiple {
         state: &mut MutexGuard<'_, ConnectionManagerMultipleState>,
         conn: ElectrumConnection,
     ) -> Result<(), ConnectionManagerErr> {
-        let (_, conn_ctx) = Self::get_connection_ctx_mut(state, &conn.addr)?;
+        let (_, conn_ctx) = state.get_connection_ctx_mut(&conn.addr)?;
         conn_ctx.connection.replace(Arc::new(AsyncMutex::new(conn)));
         Ok(())
```

Comment:
This one for example, can also fit `ConnectionManagerMultipleState`

Reply:
done